client/launchpad/hooks/use-complete-launchpad-tasks-with-notice.ts

```diff
@@ -0,0 +1,60 @@
+import page from '@automattic/calypso-router';
+import { useLaunchpad, updateLaunchpadSettings } from '@automattic/data-stores';
+import { useI18n } from '@wordpress/react-i18n';
+import { useCallback, useMemo } from 'react';
+import { useDispatch, useSelector } from 'calypso/state';
+import { successNotice } from 'calypso/state/notices/actions';
+import { NoticeOptions } from 'calypso/state/notices/types';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+
+export const useCompleteLaunchpadTasksWithNotice = ( { notifyIfNothingChanged = true } = {} ) => {
+	const siteSlug = useSelector( getSelectedSiteSlug );
+
+	const dispatch = useDispatch();
+	const { __ } = useI18n();
+
+	const { data, refetch } = useLaunchpad( siteSlug );
```

Comment:
Thinking about it now, always grabbing the launchpad data, without and sort of "is enabled" flag, seems sort of aggressive. Even if the site was launched years ago, we're continuing to fetch on every page load forever.

This is no change from before though. We used to call `fetchLaunchpad` unconditionally in `componentDidMount`, so it's not like it's any worse than before.